src/components/data-table/ha-data-table.ts

```diff
@@ -94,6 +94,8 @@ export class HaDataTable extends LitElement {
 
   @property({ type: Boolean }) public selectable = false;
 
+  @property({ type: Boolean }) public clickable = false;
```

Comment:
Not sure about the name `clickable`, but don't really have a good alternative either.

Reply:
For me it makes sense since the event is also called `@click` and that is basically what the new property is logically tied to. Would be even nicer, if we could detect if that event callback is attached, but I did not find a way to do so.